### PR TITLE
Use absolute URIs for all links

### PR DESCRIPTION
**Pull request by ringods**
We leverage the Github feature to show an organization wide README when browsing to the organziation URI:

https://github.com/pulumiverse

This document contains relative links and Github doesn't seem to generate the correct absolute URIs for these. If you click the links when going via the organization URI, some relative links work, some don't:

❌ `Articles of Association`
❌ `who these board members are`
🟢 `logo`
🟢  `mascot`
 
If you go to the file directly:

https://github.com/pulumiverse/.github/blob/main/profile/README.md

it is the other way around:

🟢 `Articles of Association`
🟢 `who these board members are`
❌ `logo`
❌ `mascot`

Hence, I replaced all the relative URIs with absolute URIs to have working links in the browser in both cases.

Signed-off-by: Ringo De Smet <ringo@de-smet.name>

### CODE REVIEW

**Comment by ringods** [comment]
Plz only approve, but don't merge in my place. Pulumi wants to add an org readme as well and I pointed them to our broken relative links. I will merge after they saw the problem on our README.

**Review by simenandre — APPROVED** [review]

**Comment by simenandre** [comment]
> Plz only approve, but don't merge in my place. Pulumi wants to add an org readme as well and I pointed them to our broken relative links. I will merge after they saw the problem on our README.

We did talk about this, and no-one should merge others PRs when they have write access. Right?

**Review by usrbinkat — APPROVED** [review]

**Pull request merged**
